pkg/apis/cr/v1alpha1/repositoryserver_types.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2023 The Kanister Authors.
+
+Copyright 2017 The Kubernetes Authors.
```

Comment:
```suggestion
```
Do we need this?

Reply:
It is generated by code generator, so adding a copyright of kubernetes